packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -647,3 +647,791 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
+          expect($('input[name="terra-time-hour-input"]')).toHaveValue('12');
+          expect($('input[name="terra-time-minute-input"]')).toHaveValue('00');
+          expect($('input[name="terra-time-second-input"]')).toHaveValue('00');
+          expect($('button[class*="meridiem"]')).toHaveAttrContaining('class', 'is-selected');
+        });
+
+        it('in the minute input should set date to today and time to 12:00 AM', () => {
+          browser.url('/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-seconds-twelve-hour');
+          browser.refresh();
+          $('input[name="terra-date-year-input"]').click();
+          browser.keys('2016');
+          $('input[name="terra-date-month-input"]').click();
+          browser.keys('05');
+          $('input[name="terra-time-minute-input"]').click();
+          browser.keys('a');
+
+          const today = moment.utc();
+          expect($('input[name="terra-date-year-input"]')).toHaveValue(today.year().toString());
+          expect($('input[name="terra-date-month-input"]')).toHaveValue((`0${(today.month() + 1)}`).slice(-2));
+          expect($('input[name="terra-date-day-input"]')).toHaveValue((`0${today.date()}`).slice(-2));
+          expect($('input[name="terra-time-hour-input"]')).toHaveValue('12');
+          expect($('input[name="terra-time-minute-input"]')).toHaveValue('00');
+          expect($('input[name="terra-time-second-input"]')).toHaveValue('00');
+          expect($('button[class*="meridiem"]')).toHaveAttrContaining('class', 'is-selected');
+        });
+
+        it('in the second input should set date to today and time to 12:00 AM', () => {
+          browser.url('/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-seconds-twelve-hour');
+          browser.refresh();
+          $('input[name="terra-date-year-input"]').click();
+          browser.keys('2016');
+          $('input[name="terra-date-month-input"]').click();
+          browser.keys('05');
+          $('input[name="terra-time-second-input"]').click();
+          browser.keys('a');
+
+          const today = moment.utc();
+          expect($('input[name="terra-date-year-input"]')).toHaveValue(today.year().toString());
+          expect($('input[name="terra-date-month-input"]')).toHaveValue((`0${(today.month() + 1)}`).slice(-2));
+          expect($('input[name="terra-date-day-input"]')).toHaveValue((`0${today.date()}`).slice(-2));
+          expect($('input[name="terra-time-hour-input"]')).toHaveValue('12');
+          expect($('input[name="terra-time-minute-input"]')).toHaveValue('00');
+          expect($('input[name="terra-time-second-input"]')).toHaveValue('00');
+          expect($('button[class*="meridiem"]')).toHaveAttrContaining('class', 'is-selected');
+        });
+      });
+
+      describe('if valid date', () => {
+        it('in the hour input should not change date and should leave it as AM', () => {
```

Comment:
These docs strings are a bit confusing. Seems like we're testing that `A` does nothing? I believe the intent is to test that if the date is a valid AM date then nothing happens? If so please change line 1228 and that will make it clear.

Reply:
Updated the doc strings to hopefully be clearer in this commit: https://github.com/cerner/terra-framework/pull/1520/commits/cf7f6e84415a91dbdd1f7d0a327de6c14e65293b